.circleci/config.yml

```diff
@@ -18,99 +17,160 @@ common: &common
         command: ./.circleci/merge_pr.sh
         when: on_fail
     - run:
-        name: merge pull request base (3nd try)
+        name: merge pull request base (3rd try)
         command: ./.circleci/merge_pr.sh
         when: on_fail
     - restore_cache:
         keys:
           - cache-v1-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "tox.ini" }}
     - run:
-        name: install unix dependencies
+        name: install dependencies
         command: |
-          sudo apt install -y build-essential
-          sudo apt install -y automake
-          sudo apt install -y pkg-config
-          sudo apt install -y libtool
-          sudo apt install -y libffi-dev
-          sudo apt install -y libgmp-dev
-          echo "Done Installing Unix Packages"
+          python -m pip install --upgrade pip
+          python -m pip install tox
     - run:
-        name: install dependencies
-        command: pip install --user tox
+        name: install pre-commit
+        command: python -m pip install --progress-bar=off pre-commit
     - run:
         name: run tox
-        command: ~/.local/bin/tox
+        command: python -m tox run -r
     - save_cache:
         paths:
           - .hypothesis
           - .tox
           - ~/.cache/pip
           - ~/.local
-          - ./eggs
+        key: cache-v1-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "tox.ini" }}
+
+orbs:
+  win: circleci/windows@5.0.0
+
+windows_steps: &windows_steps
+  executor:
+    name: win/default
+    shell: bash.exe
+  working_directory: C:\Users\circleci\project\eth-keyfile
+  steps:
+    - checkout
+    - restore_cache:
+        keys:
+          - cache-v1-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "tox.ini" }}
+    - run:
+        name: install dependencies
+        command: |
+          python -m pip install --upgrade pip
+          python -m pip install tox
+    - run:
+        name: run tox
+        command: python -m tox run -r
+    - save_cache:
+        paths:
+          - .tox
         key: cache-v1-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "tox.ini" }}
 
 jobs:
-  py37-core:
+  docs:
```

Comment:
Are there docs to come? Should this be active even though we don't (yet?) produce docs?

Reply:
the `docs` command builds the newsfragments for the changelog on non-docs libs. It's partly to make sure they build correctly, partly because it's part of the template and makes it less to manage when updating, and partly because maybe non-docs libs will get real docs someday so nice to have the plumbing in place. Is my reasons :)